frontend/src/components/maps/leaflet/InfoSlideOut/ParcelAttributes.tsx

```diff
@@ -47,38 +50,44 @@ export const ParcelAttributes: React.FC<IParcelAttributes> = ({
           />
         </OuterRow>
       </ListGroup>
-      {parcelInfo?.landLegalDescription && (
-        <ListGroup>
-          <Label className="header">Legal description</Label>
-          <OuterRow>
-            <ListGroup.Item className="legal">{parcelInfo?.landLegalDescription}</ListGroup.Item>
-          </OuterRow>
-        </ListGroup>
-      )}
-      <ListGroup>
-        <Label className="header">Valuation</Label>
-        <OuterRow>
-          <ThreeColumnItem leftSideLabel={'Assessed value:'} rightSideItem={formatAssessed} />
-        </OuterRow>
-      </ListGroup>
-      {parcelInfo?.buildings && (
-        <ListGroup>
-          <Label className="header">Associated Buildings</Label>
-          {buildingsCopy.map((building, buildingId) => (
-            <ListGroup.Item key={buildingId}>
-              <Label>{building.name}</Label>
-            </ListGroup.Item>
-          ))}
-          {parcelInfo.buildings.length > 3 && (
-            <ListGroup.Item>
-              <Label>+ {parcelInfo.buildings.length - 3} more</Label>
-            </ListGroup.Item>
+      {canViewDetails && (
+        <>
+          {parcelInfo?.landLegalDescription && (
+            <ListGroup>
+              <Label className="header">Legal description</Label>
+              <OuterRow>
+                <ListGroup.Item className="legal">
+                  {parcelInfo?.landLegalDescription}
+                </ListGroup.Item>
+              </OuterRow>
+            </ListGroup>
+          )}
+          <ListGroup>
+            <Label className="header">Valuation</Label>
+            <OuterRow>
+              <ThreeColumnItem leftSideLabel={'Assessed value:'} rightSideItem={formatAssessed} />
+            </OuterRow>
+          </ListGroup>
+          {parcelInfo?.buildings && (
+            <ListGroup>
+              <Label className="header">Associated Buildings</Label>
+              {buildingsCopy.map((building, buildingId) => (
+                <ListGroup.Item key={buildingId}>
+                  <Label>{building.name}</Label>
+                </ListGroup.Item>
+              ))}
+              {parcelInfo.buildings.length > 3 && (
+                <ListGroup.Item>
+                  <Label>+ {parcelInfo.buildings.length - 3} more</Label>
+                </ListGroup.Item>
+              )}
+            </ListGroup>
           )}
-        </ListGroup>
+          <ListGroup>
+            <ListGroup.Item>{addAssociatedBuildingLink}</ListGroup.Item>
```

Comment:
This may need to be hidden if the user doesn't have the edit_properties claim.

Reply:
hm good point

Reply:
`property-edit` or `admin-properties`